### ISSUE TITLE
p5.Vector.rotate() incorrect for angleMode(DEGREES)

### ISSUE THREAD

**Issue by shiffman** [processing/p5.js#1807]
Filing this issue in class, and I will return later today to debug and fix it.  Sample code:

```javascript
function setup() { 
  createCanvas(400, 400);
  translate(200, 200);
  angleMode(DEGREES);
  var v = createVector(100, 0);
  line(0, 0, v.x,v.y);
  v.rotate(45);
  line(0, 0, v.x,v.y);
  v.rotate(45);
  line(0, 0, v.x,v.y);
} 
```

![screen shot 2017-02-14 at 9 47 14 am](https://cloud.githubusercontent.com/assets/191758/22933767/9723a32c-f29a-11e6-9d03-0b1e52847923.png)

**Comment by shiffman** [comment]
Possibly we are using an old version of p5, will check after class, I've got to stop this now.

https://github.com/processing/p5.js/issues/1728

**Comment by shiffman** [comment]
Ok ok ok ok ok, it's fixed in 0.5.7.  Back to your regularly scheduled programming!
